docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst

```diff
@@ -42,14 +42,46 @@ Downloading a database from GitHub
 
 .. include:: ../reusables/download-github-database.rst
 
+.. _filtering-databases-and-queries-by-language:
+
+Filtering databases and queries by language
+-------------------------------------------
+
+Optionally, to see databases containing a specific language and queries written for that language, you can apply a language filter using the language selector.
+
+#. To see available language filters, in the sidebar, click the **Language** title bar.
+#. Hover over the language filter you would like to apply, then click **Select**.
+
+   .. image:: ../images/codeql-for-visual-studio-code/choose-language-filter.png
+      :width: 350
+      :alt: Screenshot of the language selector. The "Select" button for a language filter is outlined in dark orange.
+
+Creating a custom query
+------------------------
+
+You can generate a query template for a specific language from the queries panel, then write your own code to quickly create a custom query.
```

Comment:
One thing to add here: If you select an existing folder in the queries panel then the new query will be created there rather than creating a new folder.

Reply:
Ah, thanks for calling that out! I've tweaked the content to reflect that; let me know if those changes make sense to you 🙇🏼‍♂️